pyfixest/estimation/feols_.py

```diff
@@ -40,6 +40,7 @@
 )
 from pyfixest.utils.utils import get_ssc, simultaneous_crit_val
 
+prediction_type = Literal["response", "link"]
```

Comment:
Very nice, TIL! We should probably have this everywhere in the code base

Reply:
Was thinking the same! Maybe even have a separate module `literals.py` to store all literals centrally in the repository?

Reply:
Opened a PR to do this =)